athena-vertica/packaged.yaml

```diff
@@ -0,0 +1,111 @@
+Transform: AWS::Serverless-2016-10-31
```

Comment:
This file should _not_ be committed. It gets generated when you publish.

Reply:
ok